### DISCUSSION TITLE
"Your GPU driver is NEWER than what NVIDIA reports!" My version is newer but the release date is older...?

### DISCUSSION THREAD

**Discussion by johngagefaulkner** [ElPumpo/TinyNvidiaUpdateChecker#193] [Q&A] [answered]
![image](https://github.com/ElPumpo/TinyNvidiaUpdateChecker/assets/46038619/c864490d-248d-4f59-b331-afe30c7eec4c)

I'm running an NVIDIA RTX 3080 Founder's Edition and, [using their official "Advanced Driver Search" page](https://www.nvidia.com/download/find.aspx), it says the latest `DCH` Game-Ready driver version is `536.99` released on August 8th, 2023.

**However,**

![image](https://github.com/ElPumpo/TinyNvidiaUpdateChecker/assets/46038619/d31228d6-afff-43d5-a691-312726b839d3)

How am I supposedly running a newer version but the release date was 6 months ago? Not sure what's happening here.

**Comment by HawaiiBeach** [comment]
Hi @johngagefaulkner. Can you run TNUC with the `--debug` command line argument and show the debug?

**Reply by johngagefaulkner** [reply]
![image](https://github.com/ElPumpo/TinyNvidiaUpdateChecker/assets/46038619/a9774852-a7d5-4e73-935b-e482cf82a009)
Done.

**Reply by johngagefaulkner** [reply]
<img width="921" alt="image" src="https://github.com/ElPumpo/TinyNvidiaUpdateChecker/assets/46038619/9da67fa9-1e71-4760-854f-bdcca2a19ea8">
It shows this driver version everywhere and I can't find anything about it online. What's even weirder is that the not only are the driver version and release date off, the driver *seems* to be working perfectly fine. I can play all sorts of games with no FPS issues that I've noticed. It does idle pretty hot (56C) but it never reaches above 72C in full-load gaming scenarios.

**Comment by HawaiiBeach** [comment, marked as answer]
A quick google shows that the driver 545.37 is a NVIDIA Insider graphics driver, meaning it's not the same release line as GRD.

TNUC does currently not support other driver types than GRD and SD.

**Reply by johngagefaulkner** [reply]
Now to figure out how I ended up with an Insider driver 🤣 I've only ever used the NVIDIA GeForce Experience, downloaded from their official website each and every time I've needed it, to install the NVIDIA Control Panel + drivers + whatever other unnecessary things it installs.

**Reply by HawaiiBeach** [reply]
I have no clue. I recommend a clean driver install. Check the wiki on how to do this.
